app/services/manifest_builder_v3/canvas_rendering_builder.rb

```diff
@@ -42,11 +42,11 @@ def resource
         @record.respond_to?(:resource) ? @record.resource.to_model : @record
       end
 
-      # Determines if the resource be downloaded by the user
+      # Determines if the resource can be downloaded by the user
       # @return [Boolean]
       def downloadable?
-        if @record.respond_to?(:resource) || @record.is_a?(FileSet)
-          @record.resource.decorate.downloadable?
+        if @record.respond_to?(:resource)
```

Comment:
I'm not sure this logic is equivalent - before FileSets always returned true.

I honestly do not know what this downloadable check is.

Reply:
I think the logic was flawed before. It checks if the parent resource has downloadable set to "public" and whether it is in a public state. Only then does it render the download links.